modules/rn_3_13_0.adoc

```diff
@@ -55,21 +55,28 @@ The following enhancements have been made to the {productname} v2 UI.
 
 The following configuration fields have been added to {productname} {producty}.
 
-[id="oauth-reassign-configuration-field"]
-=== OAuth access token reassignment configuration field 
+[id="disable-pushes-configuration-field"]
+=== Disabling pushes to the {productname} registry configuration field
+
+In some cases, a read-only option for {productname} is not possible since it requires inserting a service key and other manual configuration changes. With the release of {productname} 3.13, a new configuration field has been added: `DISABLE_PUSHES`. 
+
+When `DISABLE_PUSHES` is set to `true`, users are unable to push images or image tags to the registry when using the CLI. Most other registry operations continue as normal when this feature is enabled by using the {productname} UI. For example, changing tags, editing a repository, robot account creation and deletion, user creation, and so on are all possible by using the UI. 
+
+When `DISABLE_PUSHES` is set to `true`, the {productname} garbage collector is disabled. As a result, when `PERMANENTLY_DELETE_TAGS` is enabled, using the {productname} UI to permanently delete a tag does not result in the immediate deletion of a tag. Instead, the tag stays in the repository until `DISABLE_PUSHES` is set to `false`, which re-enables the garbage collector. {productname} administrators should be aware of this caveat when using `DISABLE_PUSHES` and `PERMANENTLY_DELETE_TAGS` together.
```

Comment:
Same suggestion for "the tag stays in the repository" -> "the image stays in the storage"